### DISCUSSION TITLE
How do I turn off the new UI

### DISCUSSION THREAD

**Discussion by bryanrasmussen** [vercel/turborepo#8537] [Help] [answered]
### Summary

The new ui is driving me nuts. I hate it. I need to get rid of it. There should be some way to turn it off without moving to an older version. 

### Additional information

```js
it's also happening when I git push. Can I turn that off, I can accept that it works that way when I am running the server I built but because we have turbo build used now it has to ruin git for me.
```

**Comment by anthonyshew** [comment, marked as answer]
If it's not your preference, you can go back to the previous streamed logs experience using [the `ui` configuration](https://turbo.build/repo/docs/reference/configuration#ui) or [the `TURBO_UI` environment variable](https://turbo.build/repo/docs/reference/system-environment-variables).
